x/erc20/module.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 // consensusVersion defines the current x/erc20 module consensus version.
-const consensusVersion = 4
+const consensusVersion = 5
```

Comment:
This work doesn't cause breaking changes, so you don't need to increase the consensus version.